packages/api/cms-api/src/user-permissions/user-permissions.types.ts

```diff
@@ -19,13 +20,24 @@ export type PermissionsForUser =
 
 export type ContentScopesForUser = ContentScope[] | UserPermissions.allContentScopes;
 
-export interface UserPermissionsOptions {
+export interface UserPermissionsOptionsBase {
     availablePermissions?: (keyof Permission)[];
     availableContentScopes?: ContentScope[];
-    userService: UserPermissionsUserService;
+}
+export interface UserPermissionsSyncOptions extends UserPermissionsOptionsBase {
+    UserService: Type<UserPermissionsUserServiceInterface>;
+    AccessControlService?: Type<AccessControlServiceInterface>;
+}
+
+export interface UserPermissionsOptions extends UserPermissionsOptionsBase {
```

Comment:
Splitting out `userService` of `UserPermissionsOptions` is a good idea, as the service should be injected using `USER_PERMISSIONS_USER_SERVICE`.

However code working with USER_PERMISSIONS_OPTIONS now needs to use `UserPermissionsOptionsBase`, though `UserPermissionsOptions` would be better.

Could you find a more concreate name for this interface so UserPermissionsOptionsBase could be  UserPermissionsOptions?

Reply:
e5a35ae45d1b6d51bbbfe4c83b0c47f73951a686

Passed options to module are now named `UserPermissionsModuleOptions`, resulting options `UserPermissionsOptions`.